app/views/items/_date_limit.html.erb

```diff
@@ -13,42 +15,43 @@
       <div class="form-group form-inline">
         <% dfrom = params["date_from"] ? params["date_from"] : [] %>
         <div class="clearfix">
-          <%= label_tag "date_from", "Date From" %>
+          <%= label_tag "date_from", t("search.dates.date_from_label", default: "Date From") %>
           <% if date_selection?(params["date_from"], params["date_to"]) %>
-            <%= link_to search_path(clear_dates_params), class: "pull-right btn btn-default btn-xs", role: "button", aria_label: "Clear Date Filter", title: "Clear Date Filter" do %>
+            <%= link_to search_path(clear_dates_params), class: "pull-right btn btn-default btn-xs", role: "button", aria_label: t("search.dates.clear_filter", default: "Clear Date Filter"), title: "Clear Date Filter" do %>
```

Comment:
That was a question that I had...I wasn't sure if we should be doing it on the title attribute, since it sounds like screenreaders don't even typically use that?  I dunno, I am happy to go back through and add it if we think it would be helpful, but I ignored it for the purposes of this passthrough.

Reply:
The title tag is more for mouse hover so people not using screen readers can still get info about the field, so I think we should i18n that too.